oracle_fdw.c

```diff
@@ -839,6 +894,10 @@ oracleGetForeignRelSize(PlannerInfo *root, RelOptInfo *baserel, Oid foreigntable
 	/* get Oracle's (bad) estimate only if plan_costs is set */
 	if (plan_costs)
 	{
+
+		fdwState->query = createQuery(fdwState, baserel, for_update, root->query_pathkeys);
+		elog(DEBUG1, "oracle_fdw: remote query is: %s", fdwState->query);
```

Comment:
I agree with you that that is ugly. I don't know the influence of plan_costs parameter when remove it.
So, I'm happy if you do that. :)